Libs/MRML/Core/vtkMRMLSliceNode.h

```diff
@@ -418,6 +440,11 @@ class VTK_MRML_EXPORT vtkMRMLSliceNode : public vtkMRMLAbstractViewNode
   vtkMatrix4x4 *UVWToSlice;
   vtkMatrix4x4 *UVWToRAS;
 
+  std::map<vtkStdString, vtkMatrix4x4*> OrientationMatrices;
```

Comment:
if you use std::map then use a std::string instead of a vtkStdString (unless you have a good reason to do so)